packages/os/ebs-volumes.rules

```diff
@@ -1,8 +1,14 @@
 ACTION=="remove", GOTO="ebs_volumes_end"
-SUBSYSTEM!="block", GOTO="ebs_volumes_end"
-ENV{DEVTYPE}!="disk", GOTO="ebs_volumes_end"
+KERNEL!="nvme*", GOTO="ebs_volumes_end"
+ATTRS{model}!="Amazon Elastic Block Store", GOTO="ebs_volumes_end"
 
 # Follow AWS recommendation of never timing out IO on EBS volumes attached via NVMe
-KERNEL=="nvme*", ATTRS{model}=="Amazon Elastic Block Store", ATTR{queue/io_timeout}="4294967295"
+KERNEL=="nvme*", ENV{DEVTYPE}=="disk", ATTRS{model}=="Amazon Elastic Block Store", ATTR{queue/io_timeout}="4294967295"
```

Comment:
nit: restore the previous guard and omit the redundant `KERNEL` and `ATTRS{model}` checks:
```suggestion
SUBSYSTEM=="block", ENV{DEVTYPE}=="disk", ATTR{queue/io_timeout}="4294967295"
```

(Is it OK to keep the `SUBSYSTEM!="block" guard at the top? Or did that cause partitions to be skipped? Not sure why you removed it. If it works it's better to have it at the top and omit it here also.)